content/blog/convince-my-boss.md

```diff
@@ -0,0 +1,31 @@
+Title: Convince Your Boss With This Email : PyCon India 2019 Date: 2019-06-01 09:00 Category: 2019 Slug: convince-your-boss-with-this-email-pycon-india-2019 Summary: Save time composing a email to convince your boss to send you to PyCon India. We got you covered.
```

Comment:
This part needs reformatting.

Reply:
This is as it was in the other blog posts. I just copied it in. Should I add newlines or is something else missing?

Reply:
Ok I see it now Vijay.